server/cluster.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+const defaultTiKVVersion = "3.0.0"
+
+var emptyKey = []byte{}
+
+type clusterInstance struct {
+	server    *Server
+	clusterID uint64
+
+	member  *memberInstance // we only support one leader pd at this time
+	regions []*regionInstance
+	stores  []*storeInstance
+
+	idAlloc uint64
+}
+
+func (s *Server) doGetCluster(id uint64) *clusterInstance {
```

Comment:
The naming is a little bit wired. Can we just use `GetCluster`?

Reply:
I was trying to put all glue code between server and api in handler.go and conversion.go. So there are some method in handler being very simple and basically just passing arguments to server implementation. I used getCluster initially but revive complains that these two methods are confusing. After scratching my head I came up with these not good names. So consider this isn't going to be public accessible and should start with lowercase what could be a better candidate?